README.md

```diff
@@ -73,19 +72,45 @@ For example, to build the image `aws_account_id.dkr.ecr.region.amazonaws.com/my-
 
 Build your container image with:
 
-```
-$ mvn compile jib:build
+```commandline
+mvn compile jib:build
 ```
 
 Subsequent builds would usually be much faster than the initial build.
 
+### Bind to a lifecycle
+
+You can also bind `jib:build` to a Maven lifecycle such as `package` by adding the following execution to your `jib-maven-plugin` definition:
+
+```xml
+<plugin>
+  <groupId>com.google.com.tools</groupId>
+  <artifactId>jib-maven-plugin</artifactId>
+  ...
+  <executions>
+    <execution>
+      <phase>package</phase>
+      <goals>
+        <goal>build</goal>
+      </goals>
+    </execution>
+  </executions>
+</plugin>
+```
+
+Then, you can build your container image with just:
+
+```commandline
+mvn package
+```
+
 ## Extended Usage
 
 Extended configuration options provide additional options for customizing the image build.
 
 Field | Default | Description
 --- | --- | ---
-`from`|`gcr.io/distroless/java`|The base image to build your application on top of.
+from|[`gcr.io/distroless/java`](https://github.com/GoogleCloudPlatform/distroless)|The base image to build your application on top of.
```

Comment:
Just wondering: if our language is "top down" application building, do you think we should describe `from` as the base image layered underneath your application? or something?